PKGBUILD

```
@@ -0,0 +1,34 @@
+_appname=omikhleia-sile-packages
+_builddir=build
+
+pkgname=${_appname}-git
```

For consistency with Arch packaging guidelines and other future possible packages, when putting this on the AUR I'd suggest a name like this:

```suggestion
pkgname=sile-package-omikhleia-git
```